src/internet_identity/internet_identity.did

```diff
@@ -215,6 +215,17 @@ type IdentityAnchorInfo = record {
     device_registration: opt DeviceRegistrationInfo;
 };
 
+type GetDeviceCredentialsResponse = record {
+    credentials : vec DeviceCredential;
+    recovery_credentials : vec DeviceCredential;
+    recovery_phrase: bool;
```

Comment:
nit: thinking about making things a bit more correct by construction. What happens if `{ recovery_credentials = [], recovery_phrase = true }`?

Reply:
Err, I just realized there are no credentials for recovery _phrases_. So I'm guessing the recovery credentials are only for "recovery keys"?

Reply:
Yes, exactly. So your example would indicate no recovery devices but the existence of a recovery phrase.